Crash on `gr google/yamlfmt`

### aqua info

```console
$ aqua info
{
  "version": "2.45.0-8",
  "commit_hash": "4a7a9fb89458e2eacb3a78a60f13096da9518fd9",
  "os": "linux",
  "arch": "amd64",
  "pwd": "/tmp",
  "root_dir": "/home/(USER)/.local/share/aquaproj-aqua",
  "env": {
    "GITHUB_TOKEN": "(masked)"
  },
  "config_files": []
}
```


### Overview

`aqua gr google/yamlfmt` crashes.

### How to reproduce

aqua.yaml

```yaml

```

Other related code such as local Registry

```

```

Executed command and output

```console
$ aqua gr google/yamlfmt
```


### Expected behaviour

No crash, graceful error message if appropriate.

### Actual behaviour

Crash, stack trace in details below.

<details>

```
$ aqua gr google/yamlfmt
panic: runtime error: index out of range [-1]

goroutine 1 [running]:
github.com/aquaproj/aqua/v2/pkg/controller/generate-registry.(*Controller).group(0xc0000f02d0, 0xc000494310, {0x7ffd49a50bcd, 0xe}, {0xc000468870?, 0xe335c0?, 0xffffffffffffff01?})
	/home/runner/work/aqua/aqua/pkg/controller/generate-registry/group.go:273 +0x5d7
github.com/aquaproj/aqua/v2/pkg/controller/generate-registry.(*Controller).generatePackage(0xc000494310?, 0xc000162ff0?, 0xc000474288, {0x7ffd49a50bcd?, 0xc0004f4370?}, {0xc000468870?, 0xc00013be28?, 0xc0004c1796?})
	/home/runner/work/aqua/aqua/pkg/controller/generate-registry/group.go:285 +0x31
github.com/aquaproj/aqua/v2/pkg/controller/generate-registry.(*Controller).getPackageInfoWithVersionOverrides(0xc0000f02d0, {0x1035050, 0xc0000b7ec0}, 0xc000494310, {0x7ffd49a50bcd, 0xe}, 0xc000474288, 0x0?, 0xc0004f51d0)
	/home/runner/work/aqua/aqua/pkg/controller/generate-registry/version_overrides.go:135 +0x6bf
github.com/aquaproj/aqua/v2/pkg/controller/generate-registry.(*Controller).getPackageInfoMain(0xc0000f02d0, {0x1035050, 0xc0000b7ec0}, 0xc000494310, {0x7ffd49a50bcd?, 0x1?}, 0x0, 0xc0004f51d0)
	/home/runner/work/aqua/aqua/pkg/controller/generate-registry/generate.go:158 +0x62d
github.com/aquaproj/aqua/v2/pkg/controller/generate-registry.(*Controller).getPackageInfo(0x10?, {0x1035050?, 0xc0000b7ec0?}, 0x414fcb?, {0x7ffd49a50bcd?, 0xc000492270?}, 0xc0004f5298, 0x46d0e5?)
	/home/runner/work/aqua/aqua/pkg/controller/generate-registry/generate.go:112 +0x49
github.com/aquaproj/aqua/v2/pkg/controller/generate-registry.(*Controller).genRegistry(0xc0000f02d0, {0x1035050, 0xc0000b7ec0}, 0xc0004f5298, 0xc0004b11c8?, 0x410825?, {0x7ffd49a50bcd, 0xe})
	/home/runner/work/aqua/aqua/pkg/controller/generate-registry/generate.go:71 +0x65
github.com/aquaproj/aqua/v2/pkg/controller/generate-registry.(*Controller).GenerateRegistry(0xc0000f02d0, {0x1035050, 0xc0000b7ec0}, 0xc0004f5298, 0xc000494310, {0xc000492270?, 0xc0004b13b0?, 0x20?})
	/home/runner/work/aqua/aqua/pkg/controller/generate-registry/generate.go:47 +0x1a5
github.com/aquaproj/aqua/v2/pkg/cli/genr.(*command).action(0xc00012a090, 0xc0003ae040)
	/home/runner/work/aqua/aqua/pkg/cli/genr/command.go:145 +0x285
github.com/urfave/cli/v2.(*Command).Run(0xc000136000, 0xc0003ae040, {0xc0003a6640, 0x2, 0x2})
	/home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.27.5/command.go:276 +0x7e2
github.com/urfave/cli/v2.(*Command).Run(0xc0001366e0, 0xc0001340c0, {0xc000034150, 0x3, 0x3})
	/home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.27.5/command.go:269 +0xa65
github.com/urfave/cli/v2.(*App).RunContext(0xc000122000, {0x1035050, 0xc0000b7ec0}, {0xc000034150, 0x3, 0x3})
	/home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.27.5/app.go:333 +0x5a5
github.com/aquaproj/aqua/v2/pkg/cli.Run({0x1035050, 0xc0000b7ec0}, 0xc00011e000, {0xc000034150, 0x3, 0x3})
	/home/runner/work/aqua/aqua/pkg/cli/runner.go:115 +0x90d
main.core(0xc000494310, 0xc00005fa20)
	/home/runner/work/aqua/aqua/cmd/aqua/main.go:45 +0x225
main.main()
	/home/runner/work/aqua/aqua/cmd/aqua/main.go:31 +0x4c
```

</details>

### Note

_No response_

--- a/pkg/controller/generate-registry/group.go
+++ b/pkg/controller/generate-registry/group.go
@@ -274,7 +274,7 @@ func (c *Controller) group(logE *logrus.Entry, pkgName string, releases []*Relea
 		prevGroup = group
 	}
 
-	if newGroups[len(newGroups)-1].allAsset != prevGroup.allAsset {
+	if len(newGroups) == 0 || newGroups[len(newGroups)-1].allAsset != prevGroup.allAsset {
 		newGroups = append(newGroups, prevGroup)
 	}
 	group := newGroups[len(newGroups)-1]

